com.github.alainm23.byte.json

```diff
@@ -0,0 +1,61 @@
+{
+    "app-id": "com.github.alainm23.byte",
+    "base": "io.elementary.BaseApp",
+    "base-version": "juno-19.08",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.alainm23.byte",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--talk-name=org.gtk.vfs",
+        "--talk-name=org.gtk.vfs.*",
+        "--talk-name=org.gnome.SessionManager",
```

Comment:
You shouldn't be talking to session manager directly, Gio.Application probably has all the feautres you need.